llama/generation.py

```diff
@@ -102,17 +104,57 @@ def build(
     def __init__(self, model: Transformer, tokenizer: Tokenizer):
         self.model = model
         self.tokenizer = tokenizer
+        self._generate_one_token_fn = self._generate_one_token
+        self._generate_one_token_fn = torch.compile(self._generate_one_token_fn,
+                                                    backend="torchxla_trace_once", fullgraph=True)
+
+
+    def _generate_one_token(self, tokens, input_tokens, input_text_mask, cur_pos_tensor, 
+                            input_pos_tensor, output_pos_tensor, cache_kvs, temperature, top_p, logprobs, prev_pos):
```

Comment:
`prev_pos` this argument here would make dynamo retrace and recompile for every token generation.

Reply:
we should get rid of this variable - it's not really used if you look at the function details.

Reply:
I fixed this in PR:https://github.com/pytorch-tpu/llama/pull/30